examples/active.ini

```diff
@@ -1,38 +1,44 @@
 [acq_params]
-amplitude = 1.0
-p90_us = 4.8
+amplitude = 0.575
+p90_us = 1.7
 deadtime_us = 10.0
 tau_us = 3500.0
 deblank_us = 1.0
-acq_time_ms = 1024.0
-nscans = 1
+acq_time_ms = 512.0574266666666
+echo_acq_ms = 5.0
+sw_khz = 3.8999532005615936
+nscans = 2
 thermal_nscans = 3
-carrierfreq_mhz = 14.895116358000001
+carrierfreq_mhz = 14.888852134000002
```

Comment:
Don't include these types of changes in the PR. Diff against the master branch and only change things you actually mean to change